stream_chat/client.py

```diff
@@ -435,3 +436,42 @@ def list_roles(self):
         List custom roles of the app
         """
         return self.get("custom_role")
+
+    def revoke_tokens(self, before=datetime.datetime.now()):
```

Comment:
This is wrong to do since it will run in parse time and won't change. Accept `None` and handle accordingly.

See https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument for the reason

Reply:
refactored this.